resources/setups/baseline/module_config.json

```diff
@@ -14,7 +14,21 @@
       "enabled": true
     },
     "switch": {
-      "enabled": true
+      "enabled": true,
```

Comment:
what's this change for?  I understand that it's inlining the module_manifest.json file, which might be a good thing -- I'm just trying to understand the motivation..

Reply:
You mentioned earlier (https://github.com/faucetsdn/daq/pull/625#discussion_r486493462 ) that module_manifest mapping shouldn't use hardcoded path, so I thought why not merge module_manifest into module_config. I think the end result is a lot cleaner.

Reply:
There is a semantic difference between them -- one is configuration that describes how a particular instance of something is, while the other is basic information about the test (but is conceptually immutable, unlike configuration).  It might be cleaner form a "it's just one file" perspective but it seems to me it's breaking abstractions.

Reply:
I think the manifests should go in the module_config as they're really the configurations for the descriptions of tests.